lib/tasks/knows_what_exercises_are_called_and_where_they_live_in_the_directory_structure.rb

```diff
@@ -0,0 +1,34 @@
+class Exercise
```

Comment:
Not sure if this was the right jump to make, but it feels "less icky" to me than before.

Exercises know where their files are and the test runner uses an exercise and test options to run the exercise tests.